graylog2-web-interface/packages/eslint-config-graylog/package.json

```diff
@@ -13,15 +13,15 @@
   "author": "Graylog, Inc. <hello@graylog.com>",
   "license": "MIT",
   "dependencies": {
-    "eslint": "^3.16.1",
-    "eslint-config-airbnb": "^14.1.0",
+    "eslint": "^4.3.0",
+    "eslint-config-airbnb": "^15.1.0",
```

Comment:
Do you know what changed there? I couldn't find any meaningful changelog. I only see more and more red in my editor, whenever I open a file, which is starting to be annoying for me :(